### PR TITLE
添加三方客户端实现AprilComment的简单介绍

### PR DESCRIPTION
**Pull request by innc11**
Mister-Hope，不好意思哈，我刚刚才看到您在AprilComment文档下的留言（因为文档站点没有开启回复提醒）

我已经在ecosystem中添加了AprilComment的简单介绍，不知道这样子写，格式行不行哈😉

### CODE REVIEW

**Comment by Mister-Hope** [comment]
Could you please add the english docs as well? Thanks!

**Comment by innc11** [comment]
英文版写好啦

File: docs/advanced/ecosystem.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Waline 是经典的前后端分离 Client/Server 架构，提供了较为完善
 
   ```html
   <script src="https://cdn.jsdelivr.net/npm/minivaline/dist/MiniValine.min.js"></script>
-
+
```
Comment (Mister-Hope) [inline docs/advanced/ecosystem.md:17]: Prettier will fail on this line.

File: docs/advanced/ecosystem.md [hunk 2]
```diff
@@ -24,6 +24,26 @@ Waline 是经典的前后端分离 Client/Server 架构，提供了较为完善
...
+  
+  <div id="comment-widget"></div>
+  <script>
+      $(function() {
+          new AprilComment({
+              el: 'comment-widget',
+              api: 'https://your-waline-backend-url.app'
+          })
+      })
+  </script>
```
Comment (Mister-Hope) [inline docs/advanced/ecosystem.md:44]: I think prettier will also fail here, two space indent is required and a ending `;` in each statement.

File: docs/en/advanced/ecosystem.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ Waline is a classic Client/Server architecture, which provides a relatively comp
 
   ```html
   <script src="https://cdn.jsdelivr.net/npm/minivaline/dist/MiniValine.min.js"></script>
-
+
```
Comment (Mister-Hope) [inline docs/en/advanced/ecosystem.md:17]: Same

File: docs/en/advanced/ecosystem.md [hunk 4]
```diff
@@ -25,6 +25,25 @@ Waline is a classic Client/Server architecture, which provides a relatively comp
...
+  <div id="comment-widget"></div>
+  <script>
+      $(function() {
+          new AprilComment({
+              el: 'comment-widget',
+              api: 'https://your-waline-backend-url.app'
+          })
+      })
+  </script>
+  ```
```
Comment (Mister-Hope) [inline docs/en/advanced/ecosystem.md:45]: Same

File: docs/en/advanced/ecosystem.md [hunk 5]
```diff
@@ -25,6 +25,25 @@ Waline is a classic Client/Server architecture, which provides a relatively comp
   </script>
   ```
 
+- [AprilComment](https://github.com/innc11/AprilComment)
+
+  By [innc11](https://github.com/innc11/AprilComment),  green-by-white theme, simple and easy,  support Markdown as the format of comment. Usage:
+
+  ```html
+  <script src="https://cdn.jsdelivr.net/npm/april-comment@latest/dist/index.js"></script>
+  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/april-comment@latest/dist/index.css" />
```
Comment (Mister-Hope) [inline docs/en/advanced/ecosystem.md:34]: Check whether this line is longger than 80

**Review by Mister-Hope — CHANGES_REQUESTED** [review]
Some optional changes.

Acceptable if you don't want to change yourself, I can add an commit fixing it after it's merged

File: docs/en/advanced/ecosystem.md (hunk 5 above)
Comment (innc11) [reply docs/en/advanced/ecosystem.md:34]: 请问像这样断行可以么？

```html
<script src="https://cdn.jsdelivr.net/npm/april-comment@latest/dist/index.js"
></script>
<link rel="stylesheet" type="text/css" 
href="https://cdn.jsdelivr.net/npm/april-comment@latest/dist/index.css" />
```

Comment (Mister-Hope) [reply docs/en/advanced/ecosystem.md:34]: https://prettier.io/playground/#N4Igxg9gdgLgprEAuEIA0IIAcYEtoDOyoAhgE5kQDuACuQkSiQDZUkCeRGARmSWAGs4MAMpZ+uKAHNkMMgFc4GOAFtucACYbNAGRLT5JKXABiEMipIw805CBLyYEdCAAWMFcwDqr3PALiYHAiDH64AG5+7HZgBFwgkgRwZDA0fFKWyABmLEkYAFYEAB4AQnyCwiIkKnA6knDZuUoghUUiklLMcACK8hDwjcx5IOJkSWR2lmQCGtRQLlhkkjBeuBowrsgAHAAMGIsQSV58WHaLcOPhDRgAjn3wadiM9gQAtFBwmpouZHB3uL80kZMkgckNmkkVLhZAoIR0ur1+g1QU0MDASNxVutNkgAExovi4ZgdADCEBUIJAFwArC55EkACoYxhg4bhRQASSg2lgIjASxwAEFuSIYOwuoMkgBfKVAA

You can paste content here if you don't want to install and commit locally

Comment (Mister-Hope) [reply docs/en/advanced/ecosystem.md:34]: If you cloned and installed projects locally, a git hook will be automatically added to make sure your code is linted when you commit.

Comment (innc11) [reply docs/en/advanced/ecosystem.md]: 谢啦，还有这么方便的在线编辑器。

我已经重写修好了格式，希望一次性通过检查 😄

Comment (Mister-Hope) [reply docs/en/advanced/ecosystem.md]: Sorry for this:

https://github.com/walinejs/waline/blob/d990afe7769d8145357da410f85d9bac6a931cef/package.json#L90-L92

You should reformat the docs clicking 

image

This is something I missed

Comment (innc11) [reply docs/en/advanced/ecosystem.md]: 现在已经改好啦

**Pull request merged**